types/node/child_process.d.ts

```diff
@@ -110,7 +124,13 @@ declare module "child_process" {
         windowsVerbatimArguments?: boolean;
     }
 
+    interface SpawnOptionsWithoutStdio extends SpawnOptions {
+        stdio?: 'pipe' | Array<undefined | 'pipe'>;
+    }
+
+    function spawn(command: string, options?: SpawnOptionsWithoutStdio): ChildProcessWithoutNullStreams;
```

Comment:
I think at least one of the two `spawn` overloads should have the second argument mandatory. Same applies to the other variant two lines below.